spec/system/internal/admin_awards_badges_spec.rb

```diff
@@ -4,9 +4,10 @@
   let(:admin) { create(:user, :super_admin) }
   let(:user) { create(:user) }
   let(:user2) { create(:user) }
+  let(:badges) { Badge.all.map { |b| CGI.escapeHTML(b.title) } }
```

Comment:
Could be `Badge.pluck(:title).map { |t| CGI.escapeHTML(t) }` for a faster query.

Reply:
Nice catch. It can actually just be `Badge.pluck(:title)` :heart: